packages/datetime/src/_datepicker.scss

```diff
@@ -97,6 +97,14 @@ $header-margin: ($header-height - $pt-input-height) / 2 !default;
       color: $pt-text-color-disabled;
     }
 
+    &.DayPicker-Day--isToday {
+      span {
```

Comment:
we try to avoid styling tag names. please create a new class instead... 

```ts
const DATEPICKER_DAY_WRAPPER = `${DATEPICKER}-day-wrapper`;
```

and if you make this a `div`, it will get the right block styling to take up the full date box

Reply:
Done